README.md

```
@@ -178,6 +178,8 @@ Create [new credentials](https://console.cloud.google.com/apis/credentials):
     2. In the `Test users` section, click `+Add users`
     3. Enter your email and press `Save`
 
+6.  Enable Google People API in https://console.cloud.google.com/marketplace/product/google/people.googleapis.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)